http-client/Network/HTTP/Client/Body.hs

```diff
@@ -52,6 +53,15 @@ brReadSome brRead' =
 brEmpty :: BodyReader
 brEmpty = return S.empty
 
+constBodyReader :: [S.ByteString] -> IO BodyReader
+constBodyReader input = do
+  iinput <- newIORef input
+  ioutput <- newIORef []
```

Comment:
You are right. It is never used. I started from a copy of dummyConnection ioutput was accidentally left in there.